projects/ng-swiper/src/lib/swiper/swiper.component.ts

```
@@ -0,0 +1,98 @@
+    currentOptions = computed<SwiperOptions>(() => {
+        let opts = this.options() || this.optionsGlobal;
+        opts = mapOptions(opts, {
+            navigation: {
+                prevEl: '.swiper-button-prev',
+                nextEl: '.swiper-button-next',
+            },
+            pagination: {
+                el: '.swiper-pagination',
+            },
+            thumbs: {
+                el: '.' + opts.thumbs?.thumbsContainerClass || '.swiper-thumbs',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)